StreamChat.xcodeproj/project.pbxproj

```diff
@@ -4377,6 +4428,15 @@
 			path = ChatMessage;
 			sourceTree = "<group>";
 		};
+		E74DB0F32656729A00508D22 /* UISDKdocumentation */ = {
+			isa = PBXGroup;
+			children = (
+				E74DB0E72656680900508D22 /* UIView+Annotations.swift */,
+				E74DB10626567FFA00508D22 /* ChatChannelListItemView_Documentation_Tests.swift */,
+			);
+			path = UISDKdocumentation;
+			sourceTree = "<group>";
+		};
```

Comment:
We already have the `DocsSnippets` target? Maybe we can the generation there? Let's not mix actual UI tests with generating reference docs.

Reply:
As we discussed, the `DocSnippets` target is currently not compiling, so probably move it later?